src/interpolation/compute/compute.test.js

```diff
@@ -1,104 +1,137 @@
 import compute from './compute'
 
+const buildPlaceholder = txt => ({
```

Comment:
In my opinion, it would be better to use classes to to avoid doing this kind of thing in the tests ... As said below, it would allow type checking as well.

Reply:
I disagree about classes (we can debate more on Slack if you wish)